UnitTests/Source/UnitTest_Extensions.cpp

```diff
@@ -77,25 +77,25 @@ namespace NMR {
 			S_OK) << L"Failed to check Materials-Spec";
 		EXPECT_TRUE(bIsSupported) << L"Materials-Spec not supported";
 		errString = std::wstring(L"invalid 3MF API interface version for Materials-Spec: " + std::to_wstring(NMR_APIVERSION_INTERFACE_MATERIALSPEC));
-		EXPECT_EQ(nAPIInterfaceVersion, (DWORD)(NMR_APIVERSION_INTERFACE_MATERIALSPEC)) << errString;
+		EXPECT_EQ(nAPIInterfaceVersion, (DWORD)(NMR_APIVERSION_INTERFACE_MATERIALSPEC)) << errString.c_str();
```

Comment:
Curious what the problem was here?  Does getting the w_char* from wstring really fix the issue?